modules/installation-configuration-parameters.adoc

```diff
@@ -767,10 +767,16 @@ host must trust the certificate.
 |A map of keys and values that the installation program adds as tags to all resources that it creates.
 |Any valid YAML map, such as key value pairs in the `<key>: <value>` format. For more information about AWS tags, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[Tagging Your Amazon EC2 Resources] in the AWS documentation.
 
+[NOTE]
```

Comment:
Please add a content type at the top of this document.

Reply:
@nalhadef content type is there at line 205